README.md

```diff
@@ -1,34 +1,166 @@
 # style-guide
-This repository includes specs for byu-theme-components and other components. All byu elements should have specs in this repository.
```

Comment:
I think it might be helpful to leave in these two sentences, explaining that all BYU resources should have specs in here eventually.